application/application.go

```diff
@@ -13,6 +13,8 @@ type Application interface {
 	SpaceResources() space.ResourceRepository
 	Users() account.UserRepository
 	OauthStates() auth.OauthStateReferenceRepository
+	ExternalProviderTokens() token.ExternalProviderTokenRepository
```

Comment:
Maybe just ExternalTokens? Not sure though.. WDYT?

Reply:
The `provider` term is significant I think :)